app/src/main/java/com/duckduckgo/app/generalsettings/GeneralSettingsActivity.kt

```diff
@@ -105,6 +128,11 @@ class GeneralSettingsActivity : DuckDuckGoActivity() {
                     } else {
                         binding.autocompleteRecentlyVisitedSitesToggle.isVisible = false
                     }
+                    binding.maliciousDisabledMessage.visibility = if (it.maliciousSiteProtectionEnabled) View.GONE else View.VISIBLE
```

Comment:
nit: `binding.maliciousDisabledMessage.isVisible = !it.maliciousSiteProtectionEnabled`